.github/workflows/lint-docs.yml

```diff
@@ -15,9 +15,9 @@ jobs:
     lint-docs:
         runs-on: ubuntu-latest
         steps:
-            - uses: actions/checkout@v4
+            - uses: actions/checkout@v5
             - name: lint php documentation
-              uses: sudo-bot/action-doctum@dev
+              uses: sudo-bot/action-doctum@v5
```

Comment:
good, a new version will be out soon. TCPDF has basic docs so everything is good for now